docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -591,11 +609,26 @@ by `oe_get_report()`.
 - `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`: for raw SGX ECDSA quote generated
 by the Intel SGX SDK.
 
-For evidence data generated by `oe_get_evidence()`, the list of claims returned
+For evidence data wrapped by a header with a format ID, the list of claims returned
 by `oe_verify_evidence()` will contain one claim named
-`OE_CLAIM_CUSTOM_CLAIMS_ID` if the evidence data contains a non-empty custom
-claims buffer. For an SGX ECDSA report or a raw SGX ECDSA quote,
-the list of claims will contain one claim named `OE_CLAIM_SGX_REPORT_DATA_ID`.
+`OE_CLAIM_CUSTOM_CLAIMS` if the evidence data contains a non-empty custom
+claims buffer. For evidence data without an embedded format ID,
+e.g. an SGX ECDSA report or a raw SGX ECDSA quote,
+the list of claims will contain one claim named `OE_CLAIM_SGX_REPORT_DATA`.
```

Comment:
The above statement would not be true for non-SGX TEEs, so needs to be reworded.
```suggestion
claims buffer. For evidence data containing an SGX ECDSA report or a raw SGX ECDSA quote without an embedded format ID,
the list of claims will contain one claim named `OE_CLAIM_SGX_REPORT_DATA`.
```